environment.yml

```diff
@@ -18,4 +18,4 @@ dependencies:
       - pytest-ordering
       - git+git://github.com/cta-observatory/ctapipe_io_lst@v0.5.3
       - ctaplot
-      - pyirf~=0.3.0
+      - pyirf=0.3.*
```

Comment:
Shouldn't anaconda just pass this to pip and it works?

Reply:
I thought so too and applied the same semantic but it was not recognized...